keyboards/preonic/keymaps/kjwon15/rules.mk

```diff
@@ -1,2 +1,3 @@
 SRC += muse.c
 KEY_LOCK_ENABLE = yes
+SLEEP_LED_ENABLE = no
```

Comment:
```suggestion
```

Reply:
Can I ask you why?

Reply:
It's not needed; none of the Preonic revisions have this set to `yes`.

Reply:
So should I remove it?